### PR TITLE
Suggester gui

### LINKED ISSUE #4461: Display suggestions from `DeprecatedMigrationSuggester` in the gui

### PR DESCRIPTION
**Pull request by mortalisk**
**Issue**
Resolves #4461

## Pre review checklist

- [ ] Added appropriate release note label
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Review by eivindjahren — CHANGES_REQUESTED** [review]
Nice! I had some suggestions.

File: tests/unit_tests/gui/test_gui_load.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from qtpy.QtWidgets import QDialog, QMessageBox, QWidget
 
 import ert.gui
+from ert._c_wrappers.enkf import ResConfig
```
Comment (eivindjahren) [inline tests/unit_tests/gui/test_gui_load.py:11]: isort says you should sort those.

File: src/ert/gui/gert_main.py [hunk 2]
```diff
@@ -30,10 +38,69 @@
 from ert.services import Storage
 
 
+def make_suggester(suggestions, args, res_config=None):
```
[the remark this answers is not in the document]
Comment (mortalisk) [reply src/ert/gui/gert_main.py:41]: What do you mean?

**Review by eivindjahren — APPROVED** [review]
big :brain:

**Pull request merged**